sys/include/net/ng_rpl.h

```diff
@@ -1,5 +1,6 @@
 /*
- * Copyright (C) 2015 Martine Lenders <mlenders@inf.fu-berlin.de>
+ * Copyright (C) 2013 - 2014  INRIA.
+ * Copyright (C) 2015 Cenk Gündoğan <cnkgndgn@gmail.com>
```

Comment:
@authmillenon added a stub file with no practical content. I replaced that file because of merge conflicts. I will revert this change.